CHANGELOG

```diff
@@ -268,7 +268,7 @@ Testing coverage
 * Fix part of #6550: Write tests for domain/user_jobs_one_off (#6842)
 * Fix part of #6550: Write tests for domain/collection_services (#6839)
 * Fix part of #6550: Write tests for domain/classifier_domain and domain/event_services (#6831)
-* Fix part of #4057: Add unit tests for lookupEnvs method of ExpressionSyntaxTreeService.ts (#6828)
+* Fix part of #4057: Add unit tests for lookupEnvs method of expression-syntax-tree.service.ts (#6828)
```

Comment:
Please don't change the Changelog.

Reply:
Revert the change.

Reply:
Made changes. @kevinlee12 PTAL

Reply:
@kevinlee12 build failed at lint-tests, probably because of merge conflicts. I was not able to resolve the merge conflicts, therefore I deleted my local branch, and recreated it from latest Oppia-develop and made a force push to my remote branch. 
I understand that force push is not recommended but I knew that it would work in this case.

If you think that this process won't work, feel free to mention and I will make another PR accordingly.

Thanks.

Reply:
Hi @rt4914 -- we strongly disapprove of force pushes, because they alter history (and can often make PRs harder to review). Could you please explain the "not able to resolve the merge conflicts" bit so that we can figure out what's going wrong there and help resolve that for the future?

Reply:
Hi @seanlip 
1. If I go to "Files Changed" tab it does not show any unnecessary changes.
2. If I run test cases on my local machine they all pass.
3. And when I try to check the *Details* on circle-ci regarding *lint_tests* where it was failing, it says that the test cases are failing but I am not able to check the exact reason for this failure.

Before all this, I did merge my branch with the latest oppia-develop branch and after that it started to fail in `lint_tests` but not locally.

Reply:
OK -- next time, please don't force push, ever. Instead, leave a comment along the lines you just did, and we'll investigate, because this seems like a dev workflow issue that we need to find a way to fix.  Thanks!